### PR TITLE
Release api_core 1.11.0

### PR DESCRIPTION
**Pull request by tseaver**
This pull request was generated using releasetool.

### CODE REVIEW

File: api_core/CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,22 @@
...
+
+05-15-2019 10:29 PDT
+
+### Implementation Changes
+
+### New Features
+
+- Refactor 'client_info' support. ([#7849](https://github.com/googleapis/google-cloud-python/pull/7849))
+
+### Dependencies
```
Comment (busunkim96) [inline api_core/CHANGELOG.md]: Could the empty sections be removed?

Comment (tseaver) [reply api_core/CHANGELOG.md]: 67d7a15

**Comment by tseaver** [comment]
Release is at https://github.com/googleapis/google-cloud-python/releases/tag/api_core-1.11.0

**Comment by tseaver** [comment]
[Manually released to PyPI](https://pypi.org/project/google-api-core/1.11.0/).

**Pull request merged**